ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -504,7 +504,11 @@ objects:
         cat /tmp/artifacts/installer/manifests/cluster-infrastructure-02-config.yml
 
         export KUBECONFIG=/tmp/artifacts/installer/auth/kubeconfig
-        update_image_registry &
+
+        #On 4.4 and 4.5 we don't have CSI Driver so we need to patch that image reg
```

Comment:
need to make sure the upgrade suite from 4.5 to 4.6 won't get caught.

Reply:
good point :/